soroban-env-common/Cargo.toml

```diff
@@ -12,7 +12,7 @@ rust-version = "1.63"
 
 [dependencies]
 soroban-env-macros = { version = "0.0.3" }
-stellar-xdr = { version = "0.0.1", default-features = false, features = [ "next" ] }
+stellar-xdr = { version = "0.0.1", default-features = false, features = [ "next", "base64" ] }
```

Comment:
Can you make it so the base64 feature of only activated when the std feature is enabled? Change a few lines down to this:

```toml
std = ["stellar-xdr/std", "stellar-xdr/base64"]
```